typescript/infra/scripts/helloworld/kathy.ts

```diff
@@ -33,6 +37,11 @@ async function main() {
   constMetricLabels.abacus_deployment = environment;
   const coreConfig = getCoreEnvironmentConfig(environment);
   const app = await getApp(coreConfig);
+  const multiProvider = await coreConfig.getMultiProvider();
+  const gasCalc = InterchainGasCalculator.fromEnvironment(
+    environment,
+    multiProvider as any,
```

Comment:
I take it this is because of the `MultiProvider<never>` issue you shared on discord?

Reply:
Indeed